include/swift/Localization/LocalizationFormat.h

```diff
@@ -154,14 +155,20 @@ class SerializedLocalizationWriter {
 };
 
 class LocalizationProducer {
+  llvm::Optional<llvm::StringSaver> localizationSaver;
+  bool printDiagnosticName;
+
 public:
+  LocalizationProducer(llvm::Optional<llvm::StringSaver> localizationSaver =
+                           llvm::Optional<llvm::StringSaver>(),
```

Comment:
I wanted to make `localizationSaver` has a default value that's why I used an `LLVM::Optional`. However, the problem from my point of view is that the `saver` will get copied and I can't move it by reference if I want it to have a default value. Any suggestions or this is good enough?